<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotionControlForm</class>
 <widget class="QWidget" name="MotionControlForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>591</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" rowstretch="4,1">
    <item row="0" column="0">
     <widget class="QTableView" name="motorTableView"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTextBrowser" name="daqOutput"/>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="initialiseButton">
        <property name="text">
         <string>Initialise</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="driveButton">
        <property name="text">
         <string>Drive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
